dashboard/config/foorm/forms/surveys/pd/csf_deep_dive_pre.0.json

```diff
@@ -0,0 +1,260 @@
+      },
+      {
+       "value": "admin_support",
+       "text": "Support from school or district administration."
+      },
+      {
+       "value": "other_teacher_support",
+       "text": "Support from other teachers in the school."
+      },
+      {
+       "value": "cs_knowledge",
+       "text": "Knowledge and understanding of computer science."
+      },
+      {
+       "value": "understanding_codeorg_tools",
+       "text": "Understanding the Code.org teaching and learning tools."
+      },
+      {
+       "value": "understanding_course_resources",
+       "text": "Understanding the course resources (lesson plans, etc)."
+      },
+      {
+       "value": "understanding_cs_pedagogy",
+       "text": "Understanding the pedagogy for a computer science classroom."
+      },
+      {
+       "value": "keeping_students_engaged",
+       "text": "Keeping students engaged during all lessons."
+      }
+     ]
+    },
+    {
+     "type": "comment",
+     "name": "other_barriers_fr",
+     "title": "What other barriers do you experience teaching CS Fundamentals?"
+    }
+   ],
+   "title": "Barriers to Teaching CS Fundamentals"
+  },
+  {
+   "name": "page2",
+   "elements": [
+    {
+     "type": "html",
+     "name": "true_of_me_now_scale_1",
+     "html": "<p>There are a variety of teaching strategies that computer science teachers use with students in classrooms.</p>\n\n<p>Below, we ask you to use a \"true of me now\" scale to self-rate how true it is that you have concrete strategies - techniques or methods you could name or describe to another teacher - for doing these things right now.</p>\n\n<table style=\"width: 100%; border: 1px solid black; margin-left: auto; margin-right: auto;” cellpadding=\"1px”><caption></caption>\n<tbody>\n<tr>\n<td style=\"width: 23%; padding-top: 3px; text-align: center; vertical-align: top;\"><strong><span style=\"font-size: 10pt;\">1<span style=\"color: #FFFFFF;\">----</span>2</span></strong></td>\n<td style=\"width: 24.7981%; text-align: center; vertical-align: top;\"><strong><span style=\"font-size: 10pt;\">3<span style=\"color: #FFFFFF;\">----</span>4<span style=\"color: #FFFFFF;\">----</span>5</span></strong></td>\n<td style=\"width: 20.2019%; text-align: center; vertical-align: top;\"><strong><span style=\"font-size: 10pt;\">6<span style=\"color: #FFFFFF;\">----</span>7</span></strong></td>\n</tr>\n<tr>\n<td style=\"width: 23%; text-align: center; vertical-align: middle;\"><span style=\"font-size: 10pt;\"><strong>Not true of me</strong></span></td>\n<td style=\"width: 24.7981%; text-align: center; vertical-align: middle;\"><span style=\"font-size: 10pt;\"><strong>Somewhat true of me now</strong></span></td>\n<td style=\"width: 20.2019%; text-align: center; vertical-align: middle;\"><span style=\"font-size: 10pt;\"><strong>Very true of me now</strong></span></td>\n</tr>\n</tbody>\n</table>"
+    },
+    {
+     "type": "matrix",
+     "name": "teaching_strategies",
```

Comment:
sort of nit: I noticed these questions (teaching_strategies and codeorg_familiarity) are in both the pre and post surveys for csf deep dive. Would you mind making the name/values the same across both surveys? Either copy yours over or update this one to match the post survey. That way it will be easy in the future if we want to use a question library instead.

Reply:
Yep good call!